### PR TITLE
Chinese translated article for Introduction from 02~06~outline

### PR DESCRIPTION
**Pull request by adadai88**
* thanks Russ merged part 01, now I submitted the remaining part of Introduction.
* revised "InnerSource" as proper noun.
* please check if my file and content is need to be revised.

### CODE REVIEW

File: introduction/zh/03-how-works.asciidoc [hunk 1]
```diff
@@ -0,0 +1,20 @@
+=== InnerSource是如何工作的?
+假设团队A使用的软件是由团队B生产的。团队A给团队B提交了一个特性需求，但团队B无法在团队A要求的时间内及时实现该功能。在InnerSource设置中，如果团队A不能得到这个特性请求，它会提一个提拉请求(Pull Request, 简称PR)代替。也就是说，团队A直接在团队B的软件中实现该功能，并提交一个带有代码更改的提拉请求。团队B合作伙伴审查并接受提交的代码。
+
+在这个例子中，我们称团队A为客人团队，称团队B为主人团队（主导团队）。客人和主人这两个术语暗示了一种类似于在家里接待客人的情况。在这种情况下，大多数人都想成为一个好主人。为了迎接客人的到来，他们确保所有的东西都保持整洁。来访者在门口受到欢迎并被邀请进来。他们可以使用家中公共区域的功能和公用设施。可能会有一些客人被要求遵守的家规。同样的，大多数客人也想表现出对家庭和主人的尊重。他们对房子里的东西很小心，在他们逗留期间遵守规则。只要他们有礼貌，他们就会希望收到回访邀请。这些围绕着家访的概念是一个隐喻，它是团队在一个宿主向另一个宿主贡献代码库时应该采取的态度和行为。
+
+让我们仔细看看InnerSource的流程机制是如何工作的。为了帮助解释，我们将列出客人和主人团队中的几个关键人物。首先，产品所有者 https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/product-owner/01-opening-article.asciidoc[_Product Owner_] 确定主人团队愿意接受哪些功能作为贡献。贡献者 https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributor_] 是客人团队中提交代码贡献以供主人团队评审的个人。受信任提交者 https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/trusted-committer/01-introduction.asciidoc[_Trusted Committer_] 代表主人团队在提供一个成功的提拉请求所需的任何及时支持和指导。在小规模的基层工作中，一个人通常同时担任产所有者和受信任的提交者的角色。
```
Comment (lenucksi) [inline introduction/zh/03-how-works.asciidoc]: Quick reminder to a future editor: These point to the English language versions, once we happen have translated the parts you are pointing to (which would be great) we'd need to update the links to point to the Chinese version too.

**Comment by rrrutledge** [comment]
Great!  Thanks, @adadai88!  Can you revert your changes to everything directly in the `introduction` folder (e.g. `introduction/01-introduction.asciidoc`)?  Those are the English articles and should remain.

File: introduction/zh/03-how-works.asciidoc (hunk 1 above)
Comment (rrrutledge) [reply introduction/zh/03-how-works.asciidoc]: Yes!  We should open an issue to track after this PR is merged.

**Comment by adadai88** [comment]
@rrrutledge @lenucksi  dear two, I would like to adopted suggest from russ,  I would close this PR, and redo it again

**Comment by rrrutledge** [comment]
Looking forward to it so much!  Thank you!

**Comment by tanzhongyibidu** [comment]
thanks， ada，you are great。

**Comment by rrrutledge** [comment]
The "redone" pull request is at https://github.com/InnerSourceCommons/InnerSourceLearningPath/pull/236.

**Pull request closed without merging**